domains/register/domains/silentlover.is-a.dev.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "github": "Silent0070"
```

```suggestion
    "username": "Silent0070"
```
This must be set to `username`.